backend/src/main/java/gov/cdc/usds/simplereport/db/model/SpecimenBodySite.java

```diff
@@ -0,0 +1,40 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Setter
+public class SpecimenBodySite extends EternalAuditedEntity {
```

Comment:
I think we want this to just be extending `AuditedEntity`

Reply:
Ah... ok.  Easy enough to change and thanks for catching this.